backend/Dockerfile

```diff
@@ -1,18 +1,24 @@
-FROM gradle:7.6.1-jdk17
+FROM eclipse-temurin:17-jdk-alpine as build
```

Comment:
Out of curiosity, do we need the full jdk here? We leverage 17-jre-alpine in the prod dockerfile, and it may save us some space here.

Reply:
this has to stay as jdk since its the build image, it needs to to compile the java code, but I changed it to jre in the run image